docscan/session/create/check/liveness.go

```diff
@@ -56,6 +56,11 @@ func (b *RequestedLivenessCheckBuilder) ForZoomLiveness() *RequestedLivenessChec
 	return b.ForLivenessType(zoom)
 }
 
+// ForZoomLiveness sets the liveness type to "STATIC"
+func (b *RequestedLivenessCheckBuilder) ForStaticLiveness() *RequestedLivenessCheckBuilder {
+	return b.ForLivenessType(static)
+}
+
```

Comment:
Accordig to [RequestedLivenessCheckBuilder](https://lampkicking.atlassian.net/browse/DOCS-1304) "manual_check": "NEVER", // optional, default to "NEVER", permit only "NEVER" is to be added to a Lveness Check. We are missing that in RequestedLivenessConfig